MetaMask/metamask-extension
# [FLASK] Add tooltips to show info about a permission

## Explanation

This adds support for showing tooltips on permissions, and adds a tooltip to all the Snaps-related permissions.

Closes #17624.

### Before



<img width="300" alt="image" src="https://user-images.githubusercontent.com/7503723/217795094-1b0fa5f9-09b7-40be-9a51-a5c4e860b9d3.png">

### After

<img width="300" alt="image" src="https://user-images.githubusercontent.com/7503723/220143125-1b6e9cd6-2688-4afe-8b05-3819ee72f870.png"> <img width="300" alt="image" src="https://user-images.githubusercontent.com/7503723/220143132-16f92257-bb98-48fc-9340-8318e85b569d.png">

## Manual Testing Steps

1. Go to `test-snaps`.
2. Click to connect to a snap.
3. Ensure that the right icon shows up, and shows a tooltip when hovering.

## Pre-merge author checklist


## Pre-merge reviewer checklist
